api/init.py

```diff
@@ -144,6 +144,13 @@ def transfer_table(data, table):
 
 
 if __name__ == "__main__":
+
+    # checks if at least 1 tab exists in the database
+    with create_app().app_context():
+        res = Table.query.get(1)
+        if res:
+            exit()
+
```

Comment:
You could move that down to the other app context, not requiring to have it open two times, when you can do it once. The loading of the json files could also be moved if you want